packages/manager/src/pages/_app.tsx

```diff
@@ -43,10 +47,30 @@ yup.addMethod(yup.string, 'alphaNumbericOnly', function noWhitespace() {
   });
 });
 
+NProgress.configure({ showSpinner: false });
```

Comment:
@akhilmhdh you can keep the default spinner it's not going to be a problem according to me

Reply:
Rest LGTM

Reply:
Spinner is removed due to navbar.